qa/tasks/cephadm.py

```diff
@@ -110,7 +110,20 @@ def download_cephadm(ctx, config, ref):
         ref = config.get('cephadm_branch', ref)
         git_url = config.get('cephadm_git_url', teuth_config.get_ceph_git_url())
         log.info('Downloading cephadm (repo %s ref %s)...' % (git_url, ref))
-        if git_url.startswith('https://github.com/'):
+        if ctx.config.get('redhat'):
```

Comment:
wondering, can we pre-install cephadm for upstream as well?

Reply:
I am not sure about this, I think this can be taken in separate PR.